notification-server/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
+## 0.3.3
+- Add latest successful processed block
```

Comment:
This is not really what happens though? 🤷 You trace when you last processed a block successfully. And I don't think this is the best solution for the purpose described in the PR description